backend/Dockerfile

```diff
@@ -1,10 +1,9 @@
 FROM node:alpine
 MAINTAINER alexlongerbeam
 WORKDIR /app
-RUN apk --no-cache add --virtual builds-deps build-base python
+RUN apk --no-cache add --virtual builds-deps build-base python postgresql
 COPY package.json .
 COPY package-lock.json .
 RUN npm install
 COPY src src
-EXPOSE 80
```

Comment:
We need it to stay with expose 80

Reply:
Can both ports be exposed?

Reply:
Only exposed 80. Fixed this in docker-compose as well